jfxtras-controls/src/main/java/jfxtras/scene/control/LocalDateTimeTextField.java

```diff
@@ -187,6 +187,36 @@ public void set(DateTimeFormatter value) {
 	public void setText(String value) { textObjectProperty.set(value); }
 	public LocalDateTimeTextField withText(String value) { setText(value); return this; }
 	
+        

```

Comment:
No problem, I will use your style. But are you sure you don't want to keep the javadoc around the getter, setter and property? 
Because if you only keep one line of Javadoc, it's cleaner in the code, but when reading it online, you end up with methods without documentation..
You tell me.